frontend/src/utils/date.ts

```diff
@@ -11,3 +11,6 @@ export const serializeDate = (date: Date = new Date()) => {
 
   throw new Error(`Invalid date: ${asString}`);
 };
+
+export const toDate = (value: Date | string) =>
```

Comment:
Do we need this function at all? Can't we handle Date and string inputs with two separate functions?
Also, we are using `date-fns` for date manipulations, so `parseISO`  or `parseJSON` can be used instead of `new Date(+value)`.
https://date-fns.org/v2.14.0/docs/parseJSON

Reply:
Reason is that I didn't want to add serial interface for state entity. Would rather just parse the object without knowing if it been serialized or not from selector perspective :)